src/Poker.js

```diff
@@ -454,13 +456,22 @@ export default class Poker extends Bot {
                 const round = nextState.rounds.pending[0];
                 const votes = round.votes;
                 const averageVote = meanBy(votes, "value");
+                const maxVote = maxBy(votes, "value");
+                const minVote = minBy(votes, "value");
+                
+                // PERT value (round to the nearest dec)
+                const suggestedVote = (minVote + 4 * averageVote + maxVote) / 6;
+                suggestedVote = Math.round(suggestedVote * 10) / 10;
+                // simplified uncertainty value to the closest hour and round
+                const voteDeviation = (maxVote - minVote) / 6
+                voteDeviation = Math.round(voteDeviation * 10) / 10;
 
                 const table = votes.reduce((table, vote) => {
                     const person = nextState.players.find(person => person.id === vote.person);
                     return Object.assign(table, { [person.firstName]: vote.value });
                 }, {});
 
-                await this.broadcast(`:high_brightness: Thank you, everyone has voted. Average vote: ${formatVote(averageVote)}\n\n${formatMarkdownTable([table])}\n\n:mega: Awaiting moderator to estimate task.`);
+                await this.broadcast(`:high_brightness: Thank you, everyone has voted.\n\nSuggested estimate: ${suggestedVote} (can take up to ${voteDeviation} more, based on vote deviation)\n\n${formatMarkdownTable([table])}\n\n:mega: Awaiting moderator to estimate task.`);
```

Comment:
Could you use the `formatVote` there for to format the suggested vote and vote deviation?